code/src/index.css

```diff
@@ -11,3 +14,153 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+header {
+  margin-left: 2%;
+  margin-right: 2%;
+}
+
+h1 {
+  font-size: 24px;
+  margin-bottom: 10px;
+}
+
+.border-line {
+  border: 1px solid #323232;
+  margin-bottom: 10px;
+}
+
+.album-list {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 1400px;
+  margin: auto;
+}
+
+.album-container {
+  display: flex;
+  flex-direction: column;
+  width: 22%;
+  box-sizing: border-box;
+  margin: 15px;
+  margin-bottom: 25px;
+}
+
+.image-container:hover .icons {
+  opacity: 1;
+}
+.image-container:hover img {
+  opacity: 0.5;
+}
+
+.play:hover {
+  transform: scale(1.2);
+}
+
+.image-container {
+  position: relative;
+}
+
+.album-cover {
+  width: 100%;
+}
+
+.icons {
+  width: 100%;
+  display: flex;
+  position: absolute;
+  opacity: 0;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  justify-content: space-around;
+  align-items: center;
+}
+
+.play {
+  height: 70px;
+  width: 70px;
+  margin: 0 15px 0 15px;
+}
+
+.heart {
+  height: 30px;
+  width: 30px;
+  margin: 0 15px 0 15px;
+}
+
+.dots {
+  height: 30px;
+  width: 30px;
+  margin: 0 15px 0 15px;
+}
+
+h2 {
+  font-family: Helvetica, sans-serif;
+  font-size: 14px;
+  margin-bottom: 5px;
+  color: #ffffff;
+}
+
+h3,
+.artist-container a {
+  display: inline;
+  font-family: Helvetica, sans-serif;
+  font-size: 14px;
+  font-weight: normal;
+  margin-top: 0px;
+  color: #a0a0a0a0;
+}
+
+.icons img {
+  opacity: 0.5;
+}
+
+.icons img:hover {
+  opacity: 1;
+}
+
+h2:hover {
+  text-decoration-line: underline;
+}
+
+h3:hover {
+  text-decoration-line: underline;
+  color: #a0a0a0a0;
```

Comment:
There is no color like that :) You probably meant `#a0a0a0;`. Basically, only 6 characters are allowed in hexadecimal approach, eventually 3 of them if we want to write shorthand